src/App.tsx

```diff
@@ -1,26 +1,164 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import { useEffect, FC, useState, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
+import { USER_ID, addTodos, delateTodos, patchTodos } from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoFiler } from './components/TodoFilter/TodoFilter';
+import { Filters } from './types/Filters';
+import { Header } from './components/Header/Header';
+import { ErrorMesages } from './components/ErrorMesages/ErrorMesagws';
+import { getActiveTodos } from './servisec/getActiveTodos';
 
-const USER_ID = 0;
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loadError, setLoadError] = useState('');
+  const [filter, setFilter] = useState(Filters.All);
+  const [loading, setLoading] = useState<number[] | null>(null);
+  const [delateTodoId, setDelateTodoId] = useState<number[] | null>(null);
```

Comment:
Check if we can use only loading state(put deleted todo id in the loading when needed)